src/components/organisms/storage/staking/Staking.tsx

```diff
@@ -138,7 +138,14 @@ const Staking: FC = () => {
 
       const stakeContract = StakingContract.getInstance(web3 as Web3)
       const receipt = await stakeContract.stake(
-        amount, { token: currency, from: account },
+        amount,
+        {
+          token: currency,
+          from: account,
+          ...parseFloat(totalStakedUSD) > 0
```

Comment:
Could we just do
```suggestion
          ...parseFloat(totalStakedUSD)
```
?

Reply:
Yes, we can. I just prefer to don't make type conversion implicitly, as you know JS is very tricky and stupid in dynamic type conversion